USBPcapCMD/cmd.c

```diff
@@ -1210,17 +1213,18 @@ int __cdecl main(int argc, CHAR **argv)
     const char *const I_long[] = {"init-non-standard-hwids", NULL};
 
     /* Extcap interface. */
-    const char *const _1_long[] = {"extcap-interfaces", NULL};
-    const char *const _2_long[] = {"extcap-interface", NULL};
-    const char *const _3_long[] = {"extcap-dlts", NULL};
-    const char *const _4_long[] = {"extcap-config", NULL};
-    const char *const _5_long[] = {"capture", NULL};
-    const char *const _6_long[] = {"fifo", NULL};
+    const char *const _1_long[] = {"extcap-version", NULL};
+    const char *const _2_long[] = {"extcap-interfaces", NULL};
+    const char *const _3_long[] = {"extcap-interface", NULL};
+    const char *const _4_long[] = {"extcap-dlts", NULL};
+    const char *const _5_long[] = {"extcap-config", NULL};
+    const char *const _6_long[] = {"capture", NULL};
+    const char *const _7_long[] = {"fifo", NULL};
 
     /* Currently ignored. */
```

Comment:
I spent some time thinking why there is such comment here... I have forgotten to remove this comment when I implemented it in 494bc83f3db254822d2830901bf415a26bc5e0e4

Reply:
I reckoned it was probably a left over.